sklearn/tree/tree.py

```diff
@@ -1628,6 +1628,19 @@ class ExtraTreeRegressor(DecisionTreeRegressor):
 
     .. [1] P. Geurts, D. Ernst., and L. Wehenkel, "Extremely randomized trees",
            Machine Learning, 63(1), 3-42, 2006.
+
+    Examples
+    --------
+    >>> from sklearn.datasets import load_boston
+    >>> from sklearn.model_selection import cross_val_score
+    >>> from sklearn.ensemble import BaggingRegressor
+    >>> from sklearn.tree import ExtraTreeRegressor
+    >>> X, y = load_boston(return_X_y=True)
+    >>> extra_tree = ExtraTreeRegressor()  # doctest: +ELLIPSIS
+    >>> reg = BaggingRegressor(extra_tree, n_estimators=10, random_state=0)
```

Comment:
I don't think it's good to wrap it with BaggingRegressor here.

Reply:
what would you do instead?

Reply:
use ExtraTreeRegressor directly since it's a small example for ExtraTreeRegressor.

Reply:
`ExtraTreeRegressor` is not supposed to be used alone, as was pointed at in https://github.com/scikit-learn/scikit-learn/pull/15213#issuecomment-541329281

Reply:
> ExtraTreeRegressor is not supposed to be used alone, as was pointed at in #15213 (comment)

thanks, ignore this review then.